docs/management/admin/host-isolation-exceptions.asciidoc

```diff
@@ -14,20 +16,28 @@ IMPORTANT: Each host isolation exception IP address should be a highly trusted a
 .. `Name your host isolation exceptions`: Enter a name to identify the host isolation exception.
 .. `Description`: Enter a description to provide more information on the host isolation exception (optional).
 .. `Enter IP Address`: Enter the IP address for which you want to allow communication with an isolated host. This must be an IPv4 address, with optional CIDR notation (for example, `0.0.0.0` or `1.0.0.0/24`, respectively).
-. Click **Add Host isolation exception**.
+. Select an option in the *Assignment* section to assign the host isolation exception to a specific integration policy:
++
+* `Global`: Assign the host isolation exception to all integration policies for {endpoint-sec}.
+* `Per Policy` (Platinum or Enterprise subscription only): Assign the host isolation exception to one or more specific {endpoint-sec} integration policies. Select each policy where you want the host isolation exception to apply.
++
+NOTE: You can also select the `Per Policy` option without immediately assigning a policy to the host isolation exception. For example, you could do this to create and review your host isolation exception configurations before putting them into action with a policy.
+. Click **Add Host isolation exception**. The new exception is added to the *Host isolation exceptions* list.
 
 [discrete]
 [[manage-host-isolation-exceptions]]
 == View and manage host isolation exceptions
 
-The **Host isolation exceptions** list displays all the host isolation exceptions that have been configured for {elastic-sec}. To refine the Host isolation exceptions list, enter a query in the search bar. You can search by name, description, or IP address.
+The **Host isolation exceptions** list displays all the host isolation exceptions that have been configured for {elastic-sec}. To refine the **Host isolation exceptions** list, enter a query in the search bar. You can search by name, description, or IP address.
```

Comment:
This might be common knowledge, but I was wondering whether it'd be worth specifying which query lang the user should use when searching for the exception. For example, would they enter an EQL query into the search bar?

Reply:
Maybe "enter a query" isn't quite right here -- it's really just a simple search. AFAIK you can't build sophisticated queries like you can in other areas of the app. @nastasha-solomon how about something like this?
```suggestion
The **Host isolation exceptions** list displays all the host isolation exceptions that have been configured for {elastic-sec}. To refine the **Host isolation exceptions** list, use the search bar to search by name, description, or IP address.
```